cmd/wait/flag.go

```diff
@@ -7,20 +7,26 @@ import (
 
 const (
 	flagKubeconfig = "kubeconfig"
+	flagProvider   = "provider"
 )
 
 type flag struct {
 	Kubeconfig string
+	Provider   string
```

Comment:
Is this already passed into the wait step? Otherwise I don't see it being wired up in the pipeline in this PR.

Reply:
> Is this already passed into the wait step? Otherwise I don't see it being wired up in the pipeline in this PR.

Pipeline would be a different repo but I have not done the full wiring yet.